src/components/base/outline-button/outline-button.stories.ts

```diff
@@ -1,62 +1,73 @@
 import { html, TemplateResult } from 'lit';
 import { ifDefined } from 'lit/directives/if-defined.js';
-// import { AllIcons } from '../outline-icon/config';
-import { argTypeTarget } from '../outline-link/config';
-import { argTypeSlotContent } from '../../base/outline-element/utils/utils';
+import {
+  argTypeSlotContent,
+} from '../outline-element/utils/utils';
 import { ButtonVariant, ButtonSize } from './outline-button';
 import './outline-button';
 
-const buttonOptions: ButtonVariant[] = [
-  'none',
-  'primary',
-  'secondary',
-  'tertiary',
-];
-
-const sizeOptions: ButtonSize[] = ['small', 'medium', 'large'];
+const buttonOptions: ButtonVariant[] = ['none', 'primary', 'secondary'];
+const buttonSizes: ButtonSize[] = ['small', 'medium', 'large'];
 
 export default {
   title: 'Navigation/Button',
   component: 'outline-button',
   argTypes: {
     ...argTypeSlotContent,
-    target: argTypeTarget,
+    target: {
+      name: 'button-target',
+      description: 'https://www.w3schools.com/tags/att_a_target.asp',
+      options: ['_blank', '_self', '_parent', '_top'],
```

Comment:
@himerus We are missing the option `(none)`
(also in `outline-link`)

Reply:
@shaal So you're just wanting/expecting the ability to unselect/remove the target attribute in the Story?

Reply:
Got it in. Also defaulted to a REAL url, and `_blank` for the target, though you CAN deselect it to change that. I wanted it to be blank page by default just because everyone is going to click that darn button/link.

Reply:
👍